service/monitor.go

```diff
@@ -0,0 +1,66 @@
+package service
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+)
+
+func (h *Service) buildClosedVenueMessage(offlinePeriodEnd time.Time, timezone *time.Location) string {
+	var sb strings.Builder
+
+	sb.WriteString("Venue is closed for delivery")
+	if !IsUnixZero(offlinePeriodEnd) {
+		timeFormatString := "{time}"
+		if !IsToday(offlinePeriodEnd, timezone) {
```

Comment:
I would say that if it is not today, don't write the offline period at all (because it's not that someone will wait for another day to order)

Reply:
People might use Bolt some time before midnight, and the full date will make it more clear.
(Also, I think it's interesting to see what time the restaurant has chosen to enter, even if I won't be waiting for it.)